archey/archey.py

```diff
@@ -874,7 +877,7 @@ def __init__(self):
             lspci_output = sorted([
                 (i.split(': ')[0].split(' ')[1], i.split(': ')[1])
                 for i in check_output(
-                    ['lspci'], universal_newlines=True
+                    ['lspci'], stderr=DEVNULL, universal_newlines=True
```

Comment:
Hey back,

May I propose you to use the `suppress_warnings` option entry instead ?
As it's a special behavior on your own, maybe it would be better if we let the other users know when an error / warning occurs...

@AdJaGu @Si13n7 Any opinion about this too ?

Reply:
Hey, I agree with you 😉 that looks special. 🤔